### PR TITLE
Changing gadi scripts in docs.

### PR DESCRIPTION
**Pull request by julesghub**
* Removing old install guide.
* Adding updated install/config file and a sample.pbs file

### CODE REVIEW

**Comment by jmansour** [comment]
Hey @julesghub, is the system petsc still being difficult?

**Comment by jmansour** [comment]
How should this be used?  Perhaps add a `--help` to the script.

**Comment by julesghub** [comment]
> Hey @julesghub, is the system petsc still being difficult?

Not sure if it's still difficult. This was when I started testing real world models a few months back. Hand cut petsc worked so I kept with it.

**Comment by julesghub** [comment]
> How should this be used? Perhaps add a `--help` to the script.

Added a `-h` with a usage explanation.

File: docs/install_guides/nci_gadi/install_gadi.sh [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+
+** To install **
+Review script details: modules, paths, repository urls / branches etc.
+ $ source <this_script_name>
+ $ install_full_stack
+see script for details, running $ install_petsc maybe required.
+
+** To run **
+Add the line
+ source <this_script_name>
```
Comment (jmansour) [inline docs/install_guides/nci_gadi/install_gadi.sh:15]: What's the story here @julesghub?  

So they have to run `source install_gadi.sh` to run UW?

**Comment by jmansour** [comment]
Also pop something into `CHANGES.md`.

File: docs/install_guides/nci_gadi/install_gadi.sh (hunk 1 above)
Comment (julesghub) [reply docs/install_guides/nci_gadi/install_gadi.sh:15]: yep - that's the idea

**Comment by julesghub** [comment]
@jmansour , this branch is updated to dev and I added something to CHANGES.md!

**Review by jmansour — APPROVED** [review]

**Pull request merged**